app/sequencescape_excel/sequencescape_excel/specialised_field/sanger_plate_id.rb

```diff
@@ -18,32 +18,34 @@ class SangerPlateId
       # rubocop:todo Metrics/PerceivedComplexity
       # rubocop:todo Metrics/MethodLength
       # rubocop:todo Metrics/AbcSize
-      def update(attributes = {}) # rubocop:todo Metrics/CyclomaticComplexity
-        return unless valid? && attributes[:aliquot].present? && foreign_barcode_format.present?
+      def update(_attributes = {}) # rubocop:todo Metrics/CyclomaticComplexity
+        return unless valid? && foreign_barcode_format.present?
+
+        well = sample_manifest_asset.asset
```

Comment:
You can actually just do asset here as SpecialisedField::Base has:
`delegate :asset, :sample, to: :sample_manifest_asset`

Reply:
Might make sense to add sample manifest to that delegate as well

Reply:
Cool thanks